<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNASeeDialog</class>
 <widget class="QDialog" name="QNASeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>571</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>137</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>137</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Writer</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>137</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QnA1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>780</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>글쓰기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewWindowLable">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>780</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
